EasyReflectometryApp/Logic/Proxies/Project.py

```diff
@@ -223,6 +223,13 @@ def _loadProject(self):
         with open(path, 'r') as xml_file:
             descr: dict = json.load(xml_file)
 
+        self.parent._model_proxy._model = None
+        self.parent._material_proxy._materials = None
+        self.parent._model_proxy._colors = None
```

Comment:
this is what we discussed earlier - setting things directly by moving up the parentage ladder should be discouraged, since it makes the code difficult to understand and maintain.
For the Qt6 implementation, please consider sending signals with slots in the main proxy.
e.g.
```python
self.model_proxy_clean_signal.emit()
```
instead to
```python
self.parent._model_proxy._model = None
```
etc.

Reply:
Thanks, I will try to keep that in mind ;)